catalogue/templates/catalogue/search.html

```diff
@@ -66,6 +69,21 @@
         </div>
       </div>
     </div>
+    <div id="feedbackModal" class="modal">
+      <div class="modal-background"></div>
+      <div class="modal-content">
+        <div class="modal-card">
+          <header class="modal-card-head">
+            <p class="modal-card-title">Feedback</p>
+            <button class="delete" aria-label="close"></button>
+          </header>
+          <form hx-post="{% url 'catalogue:search_feedback' %}" hx-target="#content" hx-swap="innerHTML">
```

Comment:
add `<section id="content" class="modal-card-body"></section>` here
remove the div
move form into the partial template